app/presenters/account_show_presenter.rb

```diff
@@ -78,7 +78,7 @@ def formatted_ipp_due_date
     I18n.l(user.pending_in_person_enrollment.due_date, format: :event_date)
   end
 
-  def formatted_nonbiometric_idv_date
+  def formatted_nonfacial_match_idv_date
```

Comment:
hm i agree that nonfacial_match is not great!

but i am a bit torn about what to call about our original idv offering. i think the term "legacy" isn't great because a lot of existing integrations are not going to update. in other circumstances i have used "base idv" (rather than defining it as a negation of a facial match). i am happy to update this to legacy to be more in line with what we currently have, but am curious if you (and others) have an opinion about updating that language overall

Reply:
I don't feel too strongly, but since it looks to me like this is an internal method used for rendering in a view, the choices partner make shouldn't affect it too much? But "Base IDV" makes sense too, I was just trying to match the descriptions we got from Product with "Legacy IDV"

Reply:
ah okay, didn't realize there was direct guidance on this from product, thanks! will update to legacy

Reply:
updated in https://github.com/18F/identity-idp/pull/11296/commits/b1b77c3019061bf2c4bd600f31e60e717cfa687e -- thanks for flagging!